solarwinds_apm/configurator.py

```diff
@@ -173,7 +172,9 @@ def _configure_otel_components(
                     apm_config,
                     oboe_api,
                 )
-                self._configure_logs_exporter(apm_config)
+
+            # Always setup logs exporter
```

Comment:
This will always be called for now. Currently `_configure_logs_exporter` includes a check of `OTEL_PYTHON_LOGGING_AUTO_INSTRUMENTATION_ENABLED` and if False does not set up log handling nor exporter. This will change in another upcoming PR.